### PR TITLE
[Tools] Enable PHPCS for generate_project_statistics_csv.php

### LINKED ISSUE #5120: [tools] Update project statistics script
the project stats script `generate_project_statistics_csv.php`, created to get numbers from other projects  : 

- [ ] Print the $projectname in the csv output file : i.e. Move lines 23 and 24 down to 121, and prefix $projectname to $output_file
- [ ] Update email address to loris.info@mcin.ca (not exposed on the public listserv loris-dev@bic.mni.mcgill.ca )
- [ ] review for other updates since 2016...

from RM 8626

### PR DESCRIPTION
**Pull request by laemtl**
Replaces #5989
Resolves #5120

### CODE REVIEW

File: tools/generate_project_statistics_csv.php [hunk 1]
```diff
@@ -1,29 +1,19 @@
...
 /**
- * @author Stella Lee
- * Date: 15-08-15
+ * Generates a CSV report of various statistics about this study.
  *
- * Project Statistics - Updating Counts From Each LORIS
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  */
+require_once 'generic_includes.php';
+require_once __DIR__ . '/../php/libraries/OutputWrapper.class.inc';
```
Comment (driusan) [inline tools/generate_project_statistics_csv.php:9]: This is in the php/libraries directory, it should be autoloaded.

Comment (laemtl) [reply tools/generate_project_statistics_csv.php]: @driusan Fixed.

**Comment by driusan** [comment]
@jesscall @sruthymathew123 this PR is more of a rewrite of a script that happens to get PHPCS to pass than running PHPCS. I noticed that neither the original PR that it replaces nor this one was reviewed by anyone on a project. Can you guys have a look and make sure it works on your respective projects since the point of the script is for projects to run it?

**Comment by jesscall** [comment]
```
$ php tools/generate_project_statistics_csv.php 

[*] Generating project statistics for LORIS Demonstration Database

[*] Project statistics generated: project_statistics_2020-08-06.csv

[*] Please email this file to loris.info@mcin.ca.

$ cat project_statistics_2020-08-06.csv 

Project,"Number of Scanning - Visits","Number of Sites","Variable Count","Number of instruments","total number of visits (for all candidates)","number of candidates","GB of imaging data (raw and processed)","number of scans"
"LORIS Demonstration Database",37,3,153,5,808,656,0.029045514,311

**Comment by ridz1208** [comment]
@christinerogers is this the expected output for this script ? (see @jesscall's comment)

**Comment by christinerogers** [comment]
> @christinerogers is this the expected output for this script ? (see @jesscall's comment)

@ridz1208 I support merging this as is -  the point of this PR is just to get it working with phpcs. I think the query updates should go in the next PR (because they'll require discussion).

that said, @laemtl there's a mistake in the PR header -- this resolves only 1 of 3 points in the issue #5120 so this shouldn't close the issue when merged.  We'll do a further PR to update the content of the queries.

**Pull request merged**